DOCSP-35722 Workaround for using find() and sort() together (#698)
mongodb/docs-app-services: source/functions/mongodb/api.txt

.. default-domain:: mongodb

.. meta:: 
   :keywords: code example

>    :description: Learn how to use Functions for querying using MongoDB's Atlas API. 

Reviewer: ```suggestion
   :description: Learn how to use Functions to query a MongoDB Atlas data source.
```

Your description is technically correct. This phrasing avoids the double "ing" + "Atlas API" on its own kind of makes me think of the Atlas Admin API instead of the "Query API"/MQL.

   :hidden:

Finds a single document from a collection or view. If multiple documents
match the query, this returns the first matching document in the collection.

.. note:: ``findOne()`` Cannot Use Sort

  As a workaround, use ``find()`` with the ``sort()`` and ``next()`` cursor methods 
  to return a single document from a sorted collection. 

  .. code-block:: javascript

>     collection.find().sort().next()

Reviewer: ```suggestion
    collection.find({}).sort({ "<field name>": -1 }).next()
```

I think `sort()` requires a sort document as its argument, so let's include one here
